cluster/addons/fluentd-elasticsearch/kibana-image/Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY:	build push
 
-TAG = 1.0
+TAG = 1.1
```

Comment:
Unfortunately, not familiar enough with Makefiles to have a suggestion. I guess it'd be nice if there was some special step to promote something to have official version tags?